File: src/main/java/com/morak/performancetracker/configuration/PerformanceConfiguration.java
package com.morak.performancetracker.configuration;

import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.EnableAspectJAutoProxy;

@Configuration
@ComponentScan(basePackages = "com.morak.performancetracker")
@EnableAspectJAutoProxy(proxyTargetClass = true)
public class PerformanceConfiguration {
}

File: src/main/kotlin/com/morak/performancetracker/configuration/PerformanceConfiguration.kt
package com.morak.performancetracker.configuration

import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.context.annotation.ComponentScan
import org.springframework.context.annotation.Configuration
import org.springframework.context.annotation.EnableAspectJAutoProxy

@Configuration
@ComponentScan(basePackages = ["com.morak.performancetracker"])
@EnableAspectJAutoProxy(proxyTargetClass = true)
@EnableConfigurationProperties(value = [DescriptorProperties::class])
class PerformanceConfiguration
